portal/common/lib/http/http_error_responses.ts

```diff
@@ -5,23 +5,27 @@ import template_404 from "../../static/404-page.template.html";
 
 export function siteNotFound(): Response {
     return Response404(
-        "You have reached the end of the internet, please turn back!" +
+        "You have reached the end of the internet, please turn back!",
         "Invalid URL: The object ID is not a valid Walrus Site."
     );
 }
 
 export function noObjectIdFound(): Response {
-    return Response404("You have reached the end of the internet, please turn back!" +
-        "Invalid URL: No object ID could be found.");
+    return Response404(
+        "You have reached the end of the internet, please turn back!",
```

Comment:
Nit: use const for this, since it is used twice

Nit: 
```suggestion
        "You have reached the end of the Internet, please turn back!",
```